app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -121,9 +118,6 @@ interface Pixel {
 
     object PixelParameter {
```

Comment:
think this whole object can be deleted now

Reply:
Oh more deletions make me happy... Done!